test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/SchemaCompare/SchemaCompareTestUtils.cs

```diff
@@ -158,24 +163,65 @@ internal static bool ValidateOptionsEqualsDefault(SchemaCompareOptionsResult opt
+                    var actualP = v.GetValue(actualOpt);
+                    var actualPValue = actualP.GetType().GetProperty("Value").GetValue(actualP);
+
+                    if (v.Name == "ExcludeObjectTypes")
+                    {
+                        Assert.True((defaultPValue as ObjectType[]).Length == (actualPValue as ObjectType[]).Length, $"Number of excluded objects is different; expected: {(defaultPValue as ObjectType[]).Length} actual: {(actualPValue as ObjectType[]).Length}");
+                    }
+                    else
+                    {
+                        // Verifying expected and actual deployment options properties are equal
+                        Assert.True((defaultPValue == null && String.IsNullOrEmpty(actualPValue as string))
+                         || (defaultPValue).Equals(actualPValue)
+                        , $"Actual Property from Service is not equal to default property for {v.Name}, Actual value: {actualPValue} and Default value: {defaultPValue}");
+                    }
                 }
             }
+
+            // Verify the default optionsMapTable with the SchemaCompareOptionsResult options property values
+            VerifyExpectedAndActualOptionsMapTables(defaultOpt.OptionsMapTable, options.DefaultDeploymentOptions.OptionsMapTable);
+
             return true;
         }
+
+        /// <summary>
+        /// Validates the DeploymentOptions optionsMapTable with the DacDeployOptions
+        /// </summary>
+        /// <param name="expectedOptionsMapTable"></param>
+        /// <param name="dacDeployOptions"></param>
+        private static void VerifyOptionsMapTable(Dictionary<string, DeploymentOptionProperty<bool>> expectedOptionsMapTable, DacDeployOptions dacDeployOptions)
+        {
+            foreach (var optionRow in expectedOptionsMapTable)
+            {
+                var dacProp = dacDeployOptions.GetType().GetProperty(optionRow.Value.PropertyName);
+                Assert.True(dacProp != null, $"DacDeploy property not present for {optionRow.Value.PropertyName}");
+                var actualValue = dacProp.GetValue(dacDeployOptions);
+                var expectedValue = optionRow.Value.Value;
+
+                Assert.AreEqual(actualValue, expectedValue, $"Actual Property from Service is not equal to default property for {optionRow.Value.PropertyName}, Actual value: {actualValue} and Default value: {expectedValue}");
+            }
+        }
+
+        /// <summary>
+        /// Verify expected and actual DeploymentOptions OptionsMapTables values
+        /// </summary>
+        /// <param name="expectedOptionsMapTable"></param>
+        /// <param name="actualOptionsMapTable"></param>
+        private static void VerifyExpectedAndActualOptionsMapTables(Dictionary<string, DeploymentOptionProperty<bool>> expectedOptionsMapTable, Dictionary<string, DeploymentOptionProperty<bool>> actualOptionsMapTable)
```

Comment:
this function looks the same as the one in the DacServiceTests.cs. Can the tests use the same helper function so the code isn't copied in 2 places?